src/index.html

```diff
@@ -11,7 +11,141 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          Name:
+          <input
+          type="text"
+          name="name"
+          autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          How old are You?
+          <input
+          type="number"
+          name="age"
+          min="1"
+          max="100"
+          value="12"
+          >
+        </label>
+        <label class="form-field">
+          Full date of birth:
+          <input type="date" name="age">
+        </label>
+        <label class="form-field">
+          I accept the term of the agreement
+          <input type="checkbox" name="terms">
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input type="password" name="password">
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-field">
+          Do you love cats?
+          <input
+          type="radio"
+          name="pet"
+          value="yes"
+          > Yes
+          <input
+          type="radio"
+          name="pet"
+          value="no"
+          > No
+        </label>
+        <label class="form-field">
+          What is your favorite color?
+          <input
+          type="color"
+          id="favcolor"
+          name="favcolor"
+          >
+        </label>
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+          type="time"
+          id="time"
+          name="time"
+          >
+        </label>
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select
+          id="cartype"
+          name="cartype"
+          multiple>
```

Comment:
```suggestion
          <select
            id="cartype"
            name="cartype"
            multiple
          >
```
Closing tag should be under the opening